lib/pf/role.pm

```diff
@@ -678,6 +659,10 @@ sub shouldAutoRegister {
             return $role;
         }
     }
+    if ($args->{'profile'}->{'_autoregister'} eq "enabled") { 
+        $logger->debug("Autoregistration set on profile " . $args->{'profile'}->getName() );
+        return 1;
```

Comment:
Well I'd say to use $TRUE even if it is inconsistent with the rest of the sub and move to them as we rewrite code.

Or if you feel like it rewrite the 0s and 1s to the constant